codice/ddf-ui
# [3.6.x] Updates summary view management

**Updates:**
-Updates management to transfer list modal
-Cleans up and fixes bugs
-Allows for extensions for great control over read only access
-Updates patch endpoint to optionally take storeIds

**To test:**
Configure some attribute aliases via the admin ui (Using Kanri, search: "alias")
Ingest some data and find it in the search. 
Click Manage Attributes for a record-
- Verify you see the attributes
- Verify you can edit some attributes
- Verify you can move attributes between lists
- Verify you can go back from the edit modal and get back to the list management view
- Verifying that saving updates the summary view as you'd expect

![Screen Shot 2020-08-18 at 10 54 57](https://user-images.githubusercontent.com/9013407/90548100-50074900-e141-11ea-8b1e-98c4c83bcae8.png)
![Screen Shot 2020-08-18 at 10 54 48](https://user-images.githubusercontent.com/9013407/90548108-5269a300-e141-11ea-8e31-f966a6f1e3f4.png)
![Screen Shot 2020-08-18 at 10 54 35](https://user-images.githubusercontent.com/9013407/90548109-5269a300-e141-11ea-89f4-d698cb0307ad.png)
![Screen Shot 2020-08-18 at 10 54 30](https://user-images.githubusercontent.com/9013407/90548111-539ad000-e141-11ea-9dd4-c15b619ba3a9.png)
